dashboard/test/ui/features/teacher_tools/level_types/bubble_choice.feature

```diff
@@ -39,7 +40,13 @@ Feature: BubbleChoice
     And I wait until element ".teacher-panel" is visible
     # Teacher has not completed level, so make sure it is not shown as complete
     Then I verify progress for the sublevel with selector ".uitest-bubble-choice:eq(0) .progress-bubble:first" is "not_tried"
-    When I click selector ".teacher-panel table td:contains(Alice)" once I see it to load a new page
+    Then I select the "New Section" option in dropdown with class "uitest-sectionselect"
+    And I wait for 5 seconds
```

Comment:
I tried everything I could to avoid this, but adding a "wait for" was the only thing I could do to get this to pass.  I added 3 seconds but got inconsistent results.